### PR TITLE
KAZOO-3217: conferences: play name on join feature

### PR DESCRIPTION
**Pull request by siplabs**

### CODE REVIEW

**Comment by jamesaimonetti** [comment]
@siplabs Haven't seen movement on this in a week. Any expectation of when this pull request will be updated?

**Comment by siplabs** [comment]
Hello! Happy new year!!!

We'll fix everything after Jan 11th. It's all because Russian annual country-wide vodka+champagne drinking contest called "Christmas holiday".

**Comment by jamesaimonetti** [comment]
@siplabs sounds like a tradition that the US could stand to import! Happy New Year!

File: applications/ecallmgr/src/ecallmgr_util.erl [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 -include("ecallmgr.hrl").
 
--define(HTTP_GET_PREFIX, "${http_get(").
+-define(HTTP_GET_PREFIX, "http_cache://").
```
Comment (siplabs) [inline applications/ecallmgr/src/ecallmgr_util.erl]: @jamesaimonetti 
tested MOH, ringback, vm, conferences... Look like it all works good.

**Comment by jamesaimonetti** [comment]
Manually merged after rebasing latest master. Thanks @siplabs !

**Comment by jamesaimonetti** [comment]
I also updated the prompt wav files, btw.

**Pull request closed without merging**